Mathlib/Combinatorics/SimpleGraph/Path.lean

```diff
@@ -1053,6 +1065,70 @@ lemma iUnion_connectedComponentSupp (G : SimpleGraph V) :
   simp only [Set.mem_range, SetLike.mem_coe]
   exact ⟨by use G.connectedComponentMk v; exact rfl, rfl⟩
 
+namespace ConnectedComponent
+
+/-- A set of vertices represents a set of components if it contains exactly
+one vertex from each component.
+-/
+structure _root_.Set.Represents (s : Set V) (C : Set (G.ConnectedComponent)) where
+  unique_rep {c : G.ConnectedComponent} (h : c ∈ C) : ∃! v, v ∈ s ∩ c.supp
+  exact {c : G.ConnectedComponent} (h : c ∉ C) : s ∩ c.supp = ∅
+
+lemma represents_of_image_exists_rep_choose (C : Set (G.ConnectedComponent)) :
+    ((fun c ↦ c.exists_rep.choose) '' C).Represents C where
```

Comment:
```suggestion
    ((fun c ↦ c.out) '' C).Represents C where
```
I think this makes a bit more sense - it should be defined as the same thing anyway

Reply:
Done:)